crates/routes/src/feeds.rs

```diff
@@ -173,6 +198,28 @@ async fn get_feed(
   )
 }
 
+fn get_limit_and_page(req: &HttpRequest) -> (i64, i64) {
```

Comment:
Could return a struct here if more query string params are added, but I think a tuple suffices for now.